firebase/src/core/data/schemas/entities-schemas/local-public-collection.json

```diff
@@ -9,6 +9,7 @@
     "memos": {
       "type": "array",
       "minItems": 1,
+      "uniqueItems": true,
```

Comment:
Does `uniqueItems` work with objects? I mean, are properties order evaluated independently? Such as `{id:1,val:a} === {val:a,id:1}`?

Reply:
Yes. They use [`fast-deep-equal`](https://github.com/epoberezkin/fast-deep-equal) to perform the comparisons